testing/test_base.sh

```diff
@@ -76,35 +76,39 @@ if [ -z `which tcpdump` ]; then
     export PATH=/usr/sbin:$PATH
 fi
 
-device_traffic="tcpdump -en -r inst/run-9a02571e8f01/scans/monitor.pcap port 47808"
-device_bcast="$device_traffic and ether broadcast"
-device_ucast="$device_traffic and ether dst 9a:02:57:1e:8f:02"
-device_xcast="$device_traffic and ether dst 9a:02:57:1e:8f:03"
-cntrlr_traffic="tcpdump -en -r inst/run-9a02571e8f02/scans/monitor.pcap port 47808"
-cntrlr_bcast="$cntrlr_traffic and ether broadcast"
-cntrlr_ucast="$cntrlr_traffic and ether dst 9a:02:57:1e:8f:01"
-cntrlr_xcast="$cntrlr_traffic and ether dst 9a:02:57:1e:8f:03"
+function test_device_traffic {
+    device_num=$1
+    peer_num=$((3-device_num))
+    device_mac=9a:02:57:1e:8f:0$device_num
+    peer_mac=9a:02:57:1e:8f:0$peer_num
+
+    device_traffic="tcpdump -en -r inst/run-9a02571e8f0$device_num/scans/monitor.pcap port 47808"
+    device_bfr_peer="$device_traffic and ether src $peer_mac and ether broadcast"
+    device_bfr3="$device_traffic and ether src 9a:02:57:1e:8f:03 and ether broadcast"
```

Comment:
nit: I'd consider extracting the 3rd MAC address to a named variable (assigned right after peer_mac) just for consistency.